code/command_block/command_block.py

```diff
@@ -0,0 +1,70 @@
+import os
+from time import sleep
+import subprocess
+import configparser
+
+lastLine = None
+sensor_logs = "foo"
```

Comment:
foo?

Reply:
Path to folder with sensor_logs, in my case folder is called foo

Reply:
I see it was solved in df5d08f9ec573d5e8aae144c7db96a12f2e98dc3